Crash in ChatBackgroundActivity.onActivityResult()

Thanks to the new crash report sharing I added to DeltaLab recently, got this crash report from an Android 10 user:
 
```
java.lang.RuntimeException: Unable to resume activity {chat.delta.lite/org.thoughtcrime.securesms.preferences.ChatBackgroundActivity}: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=11, result=-1, data=Intent { dat=content://com.android.externalstorage.documents/document/primary:DCIM/Facebook/FB_IMG_1697114050280.jpg flg=0x43 }} to activity {chat.delta.lite/org.thoughtcrime.securesms.preferences.ChatBackgroundActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4287)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4319)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2039)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7626)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:503)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Caused by: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=11, result=-1, data=Intent { dat=content://com.android.externalstorage.documents/document/primary:DCIM/Facebook/FB_IMG_1697114050280.jpg flg=0x43 }} to activity {chat.delta.lite/org.thoughtcrime.securesms.preferences.ChatBackgroundActivity}: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
	at android.app.ActivityThread.deliverResults(ActivityThread.java:4949)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4274)
	... 11 more
Caused by: java.lang.NullPointerException: Attempt to invoke interface method 'android.view.MenuItem android.view.MenuItem.setEnabled(boolean)' on a null object reference
	at org.thoughtcrime.securesms.preferences.ChatBackgroundActivity.onActivityResult(Unknown Source:29)
	at android.app.Activity.dispatchActivityResult(Activity.java:8192)
	at android.app.ActivityThread.deliverResults(ActivityThread.java:4942)
	... 12 more
```

It seems in his device `onPrepareOptionsMenu()` is not called when resuming the activity while receiving the intent response, hence `acceptMenuItem` variable is null by the time it is tried to be use in `onActivityResult()`

--- a/src/org/thoughtcrime/securesms/preferences/ChatBackgroundActivity.java
+++ b/src/org/thoughtcrime/securesms/preferences/ChatBackgroundActivity.java
@@ -38,7 +38,6 @@ public class ChatBackgroundActivity extends PassphraseRequiredActionBarActivity 
 
     Button galleryButton;
     Button defaultButton;
-    MenuItem acceptMenuItem;
     ImageView preview;
 
     String tempDestinationPath;
@@ -80,7 +79,6 @@ public class ChatBackgroundActivity extends PassphraseRequiredActionBarActivity 
         MenuInflater inflater = this.getMenuInflater();
         menu.clear();
         inflater.inflate(R.menu.chat_background, menu);
-        acceptMenuItem = menu.findItem(R.id.apply_background);
         super.onPrepareOptionsMenu(menu);
         return true;
     }
@@ -173,7 +171,6 @@ public class ChatBackgroundActivity extends PassphraseRequiredActionBarActivity 
         if (data != null && context != null && resultCode == RESULT_OK && requestCode == ApplicationPreferencesActivity.REQUEST_CODE_SET_BACKGROUND) {
             imageUri = data.getData();
             if (imageUri != null) {
-                acceptMenuItem.setEnabled(true);
                 Thread thread = new Thread(){
                     @Override
                     public void run() {

